data/_ui/interfaces.txt

```diff
@@ -23,6 +23,7 @@ color "dim" .25 .25 .25 0.
 color "dimmer" .18 .18 .18 0.
 color "faint" .1 .1 .1 0.
 color "dark" .05 .05 .05 0.
+color "yellow" .75 .75 0. 0.
```

Comment:
Could you make this specific to the plugin panel (maybe `"plugin outdated"` - if that's the use for it in this PR)?

Reply:
if that's what we normally do, sure

Reply:
The colors at this location are colors that are used across the entire game, hence the generic names.

We already have a plugin panel specific color defined further down, so I think it makes sense to specify the use case for this color in its name and define the two plugin colors together.
https://github.com/endless-sky/endless-sky/blob/98af346977f96149d477180b9b4cf1a3c6441da2/data/_ui/interfaces.txt#L174-L175

Reply:
sure

Reply:
done